packages/cli/src/active-workflow-manager.ts

```diff
@@ -488,8 +497,17 @@ export class ActiveWorkflowManager {
 				},
 			);
 
+			if (!dbWorkflow.activeVersion) {
```

Comment:
I don't see us selecting the activeVersion in the `findById` above, so won't this be always null?